sqlstats/sqlstats.go

```diff
@@ -104,28 +117,29 @@ func (t *Tracer) queryStats(query string) *queryStats {
 	t.mu.Lock()
 	defer t.mu.Unlock()
 	if t.queries == nil {
-		t.queries = make(map[string]*queryStats)
+		t.queries = make(map[string]*QueryStats)
 	}
 	stats = t.queries[query]
 	if stats == nil {
-		stats = &queryStats{query: query}
+		stats = &QueryStats{Query: query}
 		t.queries[query] = stats
 	}
 	return stats
 }
 
-func (t *Tracer) collect() (rows []queryStats) {
+func (t *Tracer) Collect() (rows []QueryStats) {
```

Comment:
Since everywhere else we pass `QueryStats` by its pointer, we should probably do that here too. `[]*QueryStats`

(Alternatively we could switch it to pass by value everywhere, but it's a fairly big struct so a pointer is fine).